voltdbclient/url.go

```diff
@@ -25,9 +26,23 @@ func parseURL(src string) (*url.URL, error) {
 			}
 		}
 	} else {
-		if u.Port() == "" {
+		if getPort(u.Host) == "" {
 			u.Host = u.Host + ":21212"
 		}
 	}
 	return u, nil
 }
+
+func getPort(host string) string{
```

Comment:
@mehmetalisavas I don't know a thing regarding official procedures, @nshi can help us with that.

To me it is okay so long we give proper credits to the authors. Something of the lines of 

```go
//This code snippet was taken from the go standard library
```

I have seen licence added on each files in the client , then in this file we should probably include the licence notice  https://golang.org/LICENSE ?

Again, I'm not a Lawyer but I like open source and as an open source author, attribution is the best thing we can do to make the world a better place 😄